ui/webui/test/check-storage

```diff
@@ -395,6 +395,37 @@ class TestStorageMountPoints(anacondalib.VirtInstallMachineCase):
         udevadm settle --timeout=120
         """)
 
+    def partition_disk(self, disk, partitions_params):
```

Comment:
It might make sense to describe what is expected in `partition_params`.

Reply:
Is simple comment fine or would you prefer a proper docstring?

Reply:
A comment is enough, thank you!